demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -68,7 +87,7 @@ protected void onCreate(Bundle savedInstanceState) {
             drmHandler = new DownloadedModularDrm() {
                 @Override
                 public KeySetId getKeySetId() {
-                    return null; //TODO: Actually download the keyset id.
+                    return KeySetId.of(offlineKeySetId);
```

Comment:
This will crash with a NPE in case the license is not downloaded yet.
Should we fail gracefully and return null here instead?

Reply:
Very true 😄 I'll change this to return null.

Reply:
75e2ad4